### PR TITLE
Culture Names

### PR DESCRIPTION
**Pull request by Gr770**
Just a check up on culture names.

### CODE REVIEW

File: localization/english/culture/cultures_l_english.yml [hunk 1]
```diff
@@ -141,7 +141,7 @@
  tamil:0 "Tamizhar"
```
Comment (Charanraja123) [inline localization/english/culture/cultures_l_english.yml:141]: Change this to just Tamizh. I made a mistake when I entered that

File: localization/english/culture/cultures_l_english.yml [hunk 2]
```diff
@@ -269,37 +269,37 @@
...
+ hindustani_collective_noun:0 "Madhyadeśīya"
+ gujarati:0 "Gurjara"
+ gujarati_collective_noun:1 "Gurjara"
+ punjabi:0 "Pāñcanada"
+ punjabi_collective_noun:1 "Pāñcanada"
+ rajput:0 "Rājapūtra"
+ rajput_collective_noun:1 "Rājapūtra"
+ sindhi:0 "Sindhī"
+ sindhi_collective_noun:1 "Sindhī"
+ marathi:0 "Mar‍hāṭī"
```
Comment (Charanraja123) [inline localization/english/culture/cultures_l_english.yml:288]: This seems dubious, I have never seen this spelling before

File: localization/english/culture/cultures_l_english.yml [hunk 3]
```diff
@@ -269,37 +269,37 @@
...
+ marathi_collective_noun:1 "Marhāṭe"
```
Comment (Charanraja123) [inline localization/english/culture/cultures_l_english.yml:289]: I have no idea what changing 'i' from 'e' achieved here. I'll bring up these issues in the sheet as well

**Review by Viridianus — APPROVED** [review]
Updated, now should be OK.

File: localization/english/culture/cultures_l_english.yml [hunk 4]
```diff
@@ -136,8 +136,8 @@
  bavarian:0 "Beiârisc"
  bavarian_collective_noun:0 "Beiâre"
  #Dravidian group
- dravidian_group:0 "Dravidian"
- dravidian_group_collective_noun:0 "Dravidians"
+ dravidian_group:0 "Tamiṟ"
```
Comment (Charanraja123) [inline localization/english/culture/cultures_l_english.yml:139]: Wait, what is this?

Comment (Viridianus) [reply localization/english/culture/cultures_l_english.yml:139]: It's what our sheet currently has there. @palfourti - is it your work?

Comment (Charanraja123) [reply localization/english/culture/cultures_l_english.yml:139]: Yeah I guess.  I just commented on it now. Just saw it.

Comment (Gr770) [reply localization/english/culture/cultures_l_english.yml:139]: Is this okay or should it still be Tamizh?

Comment (Charanraja123) [reply localization/english/culture/cultures_l_english.yml:139]: Definitely Dravidian, Classifying the whole south as Tamil (Tamir?) doesn't seem right. But lets wait for @palfourti to reply

**Comment by Gr770** [comment]
Needs a pass through for the other latin script languages

**Comment by Viridianus** [comment]
Idk, man, do you really pronounce Spanish with a as in gate not as in pan to print Spainish all the time, or does just the country's name intervene? :D

On a more serious note: different languages seem to actually have different capitalization conventions (briefly: French does not capitalize, and Spanish only capitalizes nouns but not adjectives). Should we maybe unify that?

**Comment by Viridianus** [comment]
Also, I expect some problems with French adjective endings (basically, it's our usual problem: you're kinda supposed to know when the adjective ends in a vowel... though this one can kinda be winged because I can't imagine Pdx dealing with it, either: most cultures end in a vowel or a mute -e but Baranis, Gurs, Rajputs, Tajiks, Mogyers, Ostyaks, Outremers, Zhangzhungs, Qiangs, Hans, Tuyuhuns, Jurchens, Kimeks, Môns and Butrs don't) and some _other_ problems with German culture-group NON-collective names (basically, we have to convert them into true adjectives).

**Comment by Charanraja123** [comment]
> Idk, dude, do you really pronounce Spanish with a as in gate not as in pan to print Spainish all the time, or does just the country's name intervene? 

You don't get it, It's his way of showing how painful this project is xP

>On a more serious note: different languages seem to actually have different capitalization conventions (briefly: French does not >capitalize, and Spanish only capitalizes nouns but not adjectives). Should we maybe unify that?

Ok. Really, Do people really expect this much accuracy from a mod?, when the base game doesn't do much (I assume, idk)

> Also, I expect some problems with French adjective endings (basically, it's our usual problem: you're kinda supposed to know when the adjective ends in a vowel... though this one can kinda be winged because I can't imagine Pdx dealing with it, either: most cultures end in a vowel or a mute -e but Baranis, Gurs, Rajputs, Tajiks, Mogyers, Ostyaks, Outremers, Zhangzhungs, Qiangs, Hans, Tuyuhuns, Jurchens, Kimeks, Môns and Butrs don't) and some _other_ problems with German culture-group NON-collective names (basically, we have to convert them into true adjectives).

I think this at some point reaches diminishing returns, I really do not want to delve this deep and do something that even Pdx does not care about. Nor does any other mod afaik. I understand you want languages to be perfect, but non-english users are not many. Doing so much for maybe a single digit players (most of them will understand the limitations, if explained to) just seems unnecessary for me.

Maybe atleast wait for the mod to blow up before we go so deep?

**Comment by Viridianus** [comment]
> Could you dumb it down for me?. The last point?

French adjective normally changes like this: brun (masculine singular) - brune (feminine singular) - bruns (masculine plural) - brunes (feminine plural). But, if it ends with -e (or -a - but not with -é), a new -e is not added: Catholic is catholique - catholique - catholiques - catholiques, Yoruba is yoruba - yoruba - yorubas - yorubas.

German adjectives are, well, adjectives. They usually decline (I'll spare you the details on how), and the relevant text is probably something like `in der $<culture_group_name>$en Group` - which is not really good if we have plurals like "Espanos" as a culture group's name.

**Comment by Charanraja123** [comment]
Thanks, I understood that now.
So I'm guessing a new custom loc?

**Comment by Gr770** [comment]
> Idk, dude, do you really pronounce Spanish with a as in gate not as in pan to print Spainish all the time, or does just the country's name intervene? :D

I'm just terrible at spelling. Also everyone around me calls it Español.

> On a more serious note: different languages seem to actually have different capitalization conventions (briefly: French does not capitalize, and Spanish only capitalizes nouns but not adjectives). Should we maybe unify that?

I did try to go back and 're-French' cultures that didn't have a localization. I'm down with changing the capitalization.

**Comment by Viridianus** [comment]
> Thanks, I understood that now.
> So I'm guessing a new custom loc?

As I said, we can kinda wing French (except for capitalization) because it's on Pdx to ensure the "e" never shoots (because for them it would MOST of the time be used unappropriately). German does not require a custom loc here - just a REALLY careful check/update which I'll perform at some moment.

**Comment by Charanraja123** [comment]
Cool. 
So wing French and Hardcode German.
